docs/authorization.md

```diff
@@ -230,6 +230,41 @@ i.e. a claimed resource is bound to the same maximal permission policy. Only the
 
 TBD: Example
 
+## Consumer Permission Claims Authorizer
+
+The requested resource type might be a claimed resource of a PermissionClaim in an APIExport
+referenced by an APIBinding in the workspace being accessed.
```

Comment:
accessed by whom? That's the consumer/provider question above.

Reply:
This is the regular non-virtual workspace access. That is, any non-API provider.